specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -610,35 +612,37 @@
 					"$ref": "./examples/SearchServiceCreateOrUpdateSkillset.json"
 				}
 			},
-			"description": "Creates a new Cognitive Search skillsets.",
+			"description": "Creates a new Cognitive Search skillset in an Azure Search service.",
 			"externalDocs": {
-				"url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-tutorial-blob"
-			},
-			"parameters": [{
-				"name": "skillsetsName",
-				"in": "path",
-				"required": true,
-				"type": "string",
-				"description": "The name of the skillset to create or update."
+				"url": "https://docs.microsoft.com/azure/search/cognitive-search-tutorial-blob"
 			},
-			{
-				"name": "skillset",
-				"in": "body",
-				"required": true,
-				"schema": {
-					"$ref": "#/definitions/Skillset"
+			"parameters": [
+				{
+					"name": "skillsetName",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"description": "The name of the skillset to create or update."
 				},
-				"description": "The definition of skills."
-			},
-			{
-				"$ref": "#/parameters/ClientRequestIdParameter"
-			},
-			{
-				"$ref": "#/parameters/PreferHeaderParameter"
-			},
-			{
-				"$ref": "#/parameters/ApiVersionParameter"
-			}],
+				{
+					"name": "skillset",
+					"in": "body",
+					"required": true,
+					"schema": {
+						"$ref": "#/definitions/Skillset"
+					},
+					"description": "The list of desired Cognitive Search skills in the skillset in an Azure Search service"
```

Comment:
This description is a bit confusing because it sounds as though the expected input is a JSON array, when in fact it's an object that contains an array. Maybe rephrase like this:

`The skillset containing one or more Cognitive Search skills to create or update in an Azure Search service.`

Reply:
Also, make sure all descriptions end with a period.

Reply:
Fixed